tests/Backend/ExternalBuckets/TeradataRegisterBucketTest.php

```diff
@@ -260,6 +260,29 @@ public function testRegisterExternalDB(): void
             ],
         ], $result);
 
+        // drop external bucket
         $this->_client->dropBucket($idOfBucket, ['force' => true, 'async' => true]);
+
+        // check that workspace user CANNOT READ from table in external bucket directly
+        try {
+            $db->getDb()->fetchAllAssociative(
+                sprintf(
+                    'SELECT COUNT(*) AS CNT FROM %s.%s',
+                    TeradataQuote::quoteSingleIdentifier($dbName),
+                    TeradataQuote::quoteSingleIdentifier('TEST_TABLE')
+                )
+            );
+            $this->fail('Database should not be authorized');
+        } catch (\Doctrine\DBAL\Exception\DriverException $e) {
+            // TODO check correct message
+            $this->assertMatchesRegularExpression(
+                "/Database 'TEST_EXTERNAL_BUCKETS' does not exist or not authorized/",
+                $e->getMessage(),
+            );
+        } catch (\Throwable $e) {
+            // TODO check correct message
+            dump(get_class($e));
+            dump($e->getMessage());
+        }
```

Comment:
TODO nahradit za spravnou message

Reply:
Upravil jsem na `The user does not have SELECT access to`, ktera by mela prijit...